src/OrchardCore.Modules/OrchardCore.Lists/ViewModels/ListPartFilterViewModel.cs

```diff
@@ -0,0 +1,43 @@
+namespace OrchardCore.Lists.ViewModels
+{
+    public class ListPartFilterViewModel
+    {
+        public string DisplayText { get; set; }
+
+        public string SelectedContentType { get; set; }
+
+        public bool CanCreateSelectedContentType { get; set; }
+
+        public ContentsOrder OrderBy { get; set; }
+
+        public ContentsStatus Status { get; set; }
+
+        public ContentsBulkAction BulkAction { get; set; }
+
+        public enum ContentsOrder
+        {
+            Modified,
+            Published,
+            Created,
+            Title,
+        }
+
+        public enum ContentsStatus
```

Comment:
Használhatod itt is a másik model enumját.

Reply:
igaz